src/Symfony/Component/EmojiTransliterator/Tests/EmojiTransliteratorTest.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\EmojiTransliterator\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\EmojiTransliterator\EmojiTransliterator;
+
+/**
+ * @requires extension intl
```

Comment:
Do we need to add `ext-intl` as dependency to composer.json?

Reply:
There is a TODO in the code to do it in 7.0.

In 6.3, we have `symfony/intl` depending on the new package for BC reasons, and we don't want to add a ext-intl dependency there.